docs/pages/User/GettingStarted/demo_k8s_on_desktop.md

```diff
@@ -0,0 +1,206 @@
+
+So far, KoD suppors firefox, libreoffice and blender, the latter with the capability to expoit any available NVIDIA GPUs (through the NVIDIA CUDA driver) if the remote node has this hardware.
+In any case, thanks to the massive use of templates, many more applications can be easily supported.
+
+When executing the `cloudify` command, the application will create:
+  * a `secret` containing an ssh key that allows the two application components to communicate securely;
+  * a `deployment` containing the application (e.g. blender) and the VNC server, whose `pod` will be scheduled on the remote cluster;
+  * a `ClusterIP` `service` that allows the remote `pod` to be reachable from other `pod`s in the cluster by using [K8s DNS for services](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/);
+  * a `pod` executing the VNC viewer, started in the local machine (i.e., on the same node where you run `cloudify`).
+  
+## Installation of the required software
+To install all the required software we need to follow this steps:
+
+1. [Install k3s](#install-k3s) in both clusters;
+2. [Install liqo](#install-liqo) in both clusters; 
+3. [Install KubernetesOnDesktop](#install-kubernetesondesktop) in the *local cluster*. Note that the foreign cluster simply runs a vanilla Liqo, without any other software.
+
+### Install k3s
+Assuming you already have two Linux machines (or Virtual Machines) up and running in the same LAN, we can install [k3s](https://k3s.io/) by using the official script as documented in the [K3s Quick-Start Guide](https://rancher.com/docs/k3s/latest/en/quick-start/).
+So, you only need to run the following command:
+
+```bash
+curl -sfL https://get.k3s.io | sh -
+```
+
+A tiny customization of the above default install is required for [liqo](https://liqo.io) to work.
+When the script ends, you need to modify the `/etc/systemd/system/k3s.service` file by adding the `--kube-apiserver-arg anonymous-auth=true` service execution parameter as in the following command:
+
+```bash
+sudo sed -i "s#server#server --kube-apiserver-arg anonymous-auth=true#" /etc/systemd/system/k3s.service
+```
+
+{{%expand "After this operation, your `k3s.service` file should look like the following:" %}}
+
+```
+[Unit]
+Description=Lightweight Kubernetes
+Documentation=https://k3s.io
+Wants=network-online.target
+
+[Install]
+WantedBy=multi-user.target
+
+[Service]
+Type=notify
+EnvironmentFile=/etc/systemd/system/k3s.service.env
+KillMode=process
+Delegate=yes
+# Having non-zero Limit*s causes performance problems due to accounting overhead
+# in the kernel. We recommend using cgroups to do container-local accounting.
+LimitNOFILE=1048576
+LimitNPROC=infinity
+LimitCORE=infinity
+TasksMax=infinity
+TimeoutStartSec=0
+Restart=always
+RestartSec=5s
+ExecStartPre=-/sbin/modprobe br_netfilter
+ExecStartPre=-/sbin/modprobe overlay
+ExecStart=/usr/local/bin/k3s \
+    server --kube-apiserver-arg anonymous-auth=true \
```

Comment:
if you want to run it on nVidia docker, you have to add also a `--docker` flag to enable k3s to run on docker

Reply:
Fixed